app/services/search/suggestions_builder.rb

```diff
@@ -0,0 +1,28 @@
+class Search::SuggestionsBuilder
+  RADIUS_OPTIONS = [1, 5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 70, 80, 90, 100, 200].freeze
```

Comment:
Can we reuse this constant for `VacanciesOptionsHelper::radius_filter_options` please? It's the same array.